index.html

```diff
@@ -226,13 +226,13 @@ <h4>When Was 'Peak Jennifer'?</h4>
                     <li data-aos="fade-up">
                         <figure>
                             <a target="_blank" class="link"
-                                href="https://https://github.com/sarahgrant11/World-Happiness-2020"></a>
-                            <img src="assets/baby_names_pop.png" alt="">
+                                href="https://github.com/sarahgrant11/World-Happiness-2020"></a>
+                            <img src="assets/happiness_global.png" alt="">
                             <figcaption>
                                 <div class='caption'>
-                                    <h4>World Happiness TK</h4>
+                                    <h4>Visualizing World Happiness</h4>
                                     <p>
-                                        Happiness Trends grouping data by country with sparklines.   
+                                        Global chloropleth map with yearly transitions and capital markers from flask app that pulled country data to create overall "happiness" rank.  
```

Comment:
Flask might be better capitalized. Also this is a long sentence lol

```suggestion
                                        Global chloropleth map with yearly transitions and capital markers, using country data pulled from a Flask app to create an overall "happiness" rank.
```